recipes/common/Makefile.common

```diff
@@ -156,8 +157,14 @@ install-chrome:
 		rm $(CHROME_DOWNLOAD_PATH); \
 	fi;
 
+.PHONY: check-umask
+check-umask:
+	@test "$(UMASK)" = "0022" || \
+		(echo; echo -n "Error: umask other then 0022 can cause unexpected behavour! "; \
```

Comment:
- nit: "behavior"
- Also we might as well print the umask, like "error: umask $(UMASK) will cause unexpected behavior: use umask 022"

And um...we should probably find a command that will forcibly reset the git checkout based on an updated umask, but I'm having trouble finding a nice way to do that that's sane; a brutal `rm * -rf; git reset --hard HEAD` works but...not my favorite

Reply:
We could invent something using `git ls-files --stage` that resets all the permissions in the repo

Reply:
I would do.

u=$(umask)
umask 022
rm -rf ai-lab-recipes
git checkout ...
umask $u